### PR TITLE
Maintenance: Unify examples

### PR DESCRIPTION
**Pull request by mkuczera**
As discuessed, unified real life examples and examples. Also prepare test setup

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,20 +24,50 @@
   },
   "dependencies": {},
   "peerDependencies": {
-    "react": ">= 15.4.1",
-    "react-native": ">= 0.40.0"
+    "react": "*",
+    "react-native": "*"
```
Comment (rotemmiz) [inline package.json]: The project requires RN>40 IIRC

File: playground/android/build.gradle [hunk 2]
```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.0'
+        classpath 'com.android.tools.build:gradle:2.2.3'
```
Comment (rotemmiz) [inline playground/android/build.gradle]: com.android.tools.build:gradle:2.2.3 is VERY old. Let's please move to 3.0.1 or at least stay at 2.3.0

File: playground/android/gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,6 +1,5 @@
-#Thu Feb 09 18:04:42 IST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
```
Comment (rotemmiz) [inline playground/android/gradle/wrapper/gradle-wrapper.properties]: Same here please, let's move forward, not backwards.

File: playground/android/build.gradle (hunk 2 above)
Comment (rotemmiz) [reply playground/android/build.gradle]: Detox will not work with that gradle plugin version, it requires 3.x.x

**Comment by mkuczera** [comment]
@rotemmiz i updated the PR. Thanks for reviewing ;)

**Comment by mkuczera** [comment]
Strange. Ran it this morning and everything was fine. Deleted the app from the phone and ran it again and it was not able to build. I will take this as a task to to everytime before pushing.
- Added google Repo to buildscripts
- Added package.json run-android
- Set gradle version to 4.1

**Comment by rotemmiz** [comment]
One more thing that needs to be verified is what npm packs inside the package with the new structure.
I suggest you run `npm pack` and then `tar -tf react-native-interactable-{version}.tgz`, and adjust `.npmignore` accordingly.

**Comment by rotemmiz** [comment]
Thanks!

**Pull request merged**